### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by lecourtoisflo**

### CODE REVIEW

File: sources/CMakeLists.txt [hunk 1]
```diff
@@ -7,21 +7,19 @@
...
-add_executable(DynaflowLauncher ${SOURCES} ${HEADERS})
+add_executable(DynaflowLauncher ${SOURCES})
 
 target_link_libraries(DynaflowLauncher
   PRIVATE
-  DynaflowLauncher::options
+  DynaflowLauncher::common
   Dynawo::dynawo_API_PAR
   Dynawo::dynawo_API_DYD
 )
```
Comment (gautierbureau) [inline sources/CMakeLists.txt:25]: newline

**Review by gautierbureau — CHANGES_REQUESTED** [review]

File: sources/common/include/Options.h [hunk 2]
```diff
@@ -20,8 +20,8 @@
  * @brief Options header file
  */
 
-/// @brief Namespace for options library
-namespace options {
+/// @brief Namespace for common components to dynawo-launcher
+namespace common {
```
Comment (rosiereflo) [inline sources/common/include/Options.h:24]: common seems quite generic, maybe we should put DFL somewhere?

**Review by rosiereflo — CHANGES_REQUESTED** [review]

**Comment by gautierbureau** [comment]
General questions: we don't follow dynawo coding style in this project ? (directories not with capital letter, and structure sepration between src and include)

**Comment by rosiereflo** [comment]
> General questions: we don't follow dynawo coding style in this project ? (directories not with capital letter, and structure sepration between src and include)

We should follow the same style in all our projects IMHO

**Comment by lecourtoisflo** [comment]
> > General questions: we don't follow dynawo coding style in this project ? (directories not with capital letter, and structure sepration between src and include)
> 
> We should follow the same style in all our projects IMHO

It is not always respected in dynawo (util). As for include / src separation it is for public headers and sources. We don't see it in dynawo so I wonder.

**Comment by lecourtoisflo** [comment]
> > > General questions: we don't follow dynawo coding style in this project ? (directories not with capital letter, and structure sepration between src and include)
> > 
> > 
> > We should follow the same style in all our projects IMHO
> 
> It is not always respected in dynawo (util). As for include / src separation it is for public headers and sources. We don't see it in dynawo so I wonder.

OK for name of he directories

File: sources/common/include/Options.h (hunk 2 above)
Comment (lecourtoisflo) [reply sources/common/include/Options.h:24]: DFL ?

Comment (rosiereflo) [reply sources/common/include/Options.h:24]: DynaFlow-Launcher

File: sources/CMakeLists.txt (hunk 1 above)
Comment (lecourtoisflo) [reply sources/CMakeLists.txt:25]: OK

**Comment by gautierbureau** [comment]
> > > General questions: we don't follow dynawo coding style in this project ? (directories not with capital letter, and structure sepration between src and include)
> > 
> > 
> > We should follow the same style in all our projects IMHO
> 
> It is not always respected in dynawo (util). As for include / src separation it is for public headers and sources. We don't see it in dynawo so I wonder.

I don't really mind if we do it but we have to be aware and make sure it is what we want. For folders with capitals it is only in the sources directory that is the rule, elsewhere where there is no cpp it is not. We can discuss to adapt them if need be

**Review by rosiereflo — APPROVED** [review]

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**